pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -123,6 +123,12 @@ func Add(mgr manager.Manager) error {
 	}
 	r.coreClient = coreClient
 
+	if err := openshift.ConsoleYAMLSampleExists(); err == nil {
+		openshift.CreateConsoleYAMLSamples(mgr.GetClient())
+	} else {
+		log.Info("Yaml Samples", "Console YAML sample is not added:", err)
```

Comment:
Would it make sense to skip logging if the error is _already exists_?

Reply:
modified.